src/APIFunctions/ErrorLog.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios' // what is this??
+
+export async function getAllErrorLogs () {
+  let allErrorLogs = []
+  await axios
+    .get('api/ErrorLog/getErrorLogs')
+    .then(res => {
+      allErrorLogs = res.data
+    })
+    .catch(err => {
+      allErrorLogs = err
+    })
+  return allErrorLogs
+}
+
+export async function AddErrorLog (newError) {
```

Comment:
add [jsdoc documentation](https://jsdoc.app/tags-param.html) to this function. An example you can take can be found here in [`src/APIFunctions/Event.js`](https://github.com/SCE-Development/Core-v4/blob/dfeabd591721a486e46c131c4ecb75c74e4e4935/src/APIFunctions/Event.js#L21)

Also:
```suggestion
export async function addErrorLog (newError) {
```